src/Computation/Constraints.cs

```diff
@@ -48,13 +48,13 @@ public BigInteger Evaluate(IValue value)
             : Enumerable.Empty<KeyValuePair<string, string>>();
     }
 
-    public static IModel Create<TContext>(IEnumerable<IValue> assertions)
+    public static IConstraints Create<TContext>(IEnumerable<IValue> assertions)
         where TContext : IContext, new()
     {
         var context = new TContext();
         var solver = context.CreateSolver();
         solver.Assert(assertions.Select(a => a.AsBool(context)).ToArray());
 
-        return new Model(context, solver);
+        return new Constraints(context, solver);
     }
 }
```

Comment:
On my travels I discovered that this should never really have been called `Model` since that would be an interpretation of the constraints rather than just the constraints themselves.